source/access/mfa_guide.rst

```diff
@@ -208,27 +195,61 @@ on your local machine first.
 
 .. _mfa_client_config:
 
+GUI applications with SSH connection in the background
+------------------------------------------------------
+
+Some applications such as MobaXTerm and FileZilla provide a Graphical User Interface
+(GUI) which makes them very useful when connecting to remote sites. However,
+such an application may not always prompt you to copy/paste the VSC firewall link to
+set up the necessary SSH certificate.
+
+Therefore, one way to connect to VSC is to first connect with an ssh-client 
```

Comment:
This is a bit ambiguous:
Where shall they first connect to (before connecting to VSC)? Please give a clear host address or extra explanation.

Reply:
I think now it's better